### PR TITLE
Add information for cert-manager

### PR DESCRIPTION
**Pull request by dramich**
Problem:
cert-manager is old and will be cut off soon

Solution:
Update docs to include current install instructions and instructions on
how to upgrade cert-manager to the current version

Issue: https://github.com/rancher/rancher/issues/22235

### CODE REVIEW

File: content/rancher/v2.x/en/upgrades/upgrades/ha-server-upgrade-helm/_index.md [hunk 1]
```diff
@@ -11,6 +11,10 @@ The following instructions will guide you through upgrading a high-availability
 >
 >       As of release v2.0.8, Rancher supports installation and upgrade by Helm chart, although RKE installs/upgrades are still supported as well. If you want to change upgrade method from RKE Add-on to Helm chart, follow this procedure.
 
+---
+
+>**Note:** [Let's Encrypt will be blocking cert-manager instances older than 0.8.0 starting Novermber 1st 2019.](https://community.letsencrypt.org/t/blocking-old-cert-manager-versions/98753) In oder to upgrade cert-manager to the newer version follow [these instructions.]({{< baseurl >}}/rancher/v2.x/en/cluster-admin/upgrade-cert-manager)
```
Comment (catherineluse) [inline content/rancher/v2.x/en/upgrades/upgrades/ha-server-upgrade-helm/_index.md]: November

File: content/rancher/v2.x/en/cluster-admin/upgrade-cert-manager/_index.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
+---
+title: Upgrade Cert-manager
+weight: 2040
+---
+
+[Let's Encrypt will be blocking cert-manager instances older than 0.8.0 starting Novermber 1st 2019.](https://community.letsencrypt.org/t/blocking-old-cert-manager-versions/98753) In oder to upgrade cert-manager to the newer version follow these instructions:
+
+
+```
+# Backup existing resources as a precation
```
Comment (catherineluse) [inline content/rancher/v2.x/en/cluster-admin/upgrade-cert-manager/_index.md]: Back up existing resources as a precaution

File: content/rancher/v2.x/en/installation/ha/helm-rancher/_index.md [hunk 3]
```diff
@@ -37,26 +37,48 @@ There are three recommended options for the source of the certificate.
 
 > **Important:** Due to an issue with Helm v2.12.0 and cert-manager, please use Helm v2.12.1 or higher.
```
Comment (catherineluse) [inline content/rancher/v2.x/en/installation/ha/helm-rancher/_index.md]: shoud the HA air gap docs be updated too?

File: content/rancher/v2.x/en/cluster-admin/upgrade-cert-manager/_index.md [hunk 4]
```diff
@@ -0,0 +1,47 @@
+---
+title: Upgrade Cert-manager
+weight: 2040
+---
+
+[Let's Encrypt will be blocking cert-manager instances older than 0.8.0 starting Novermber 1st 2019.](https://community.letsencrypt.org/t/blocking-old-cert-manager-versions/98753) In oder to upgrade cert-manager to the newer version follow these instructions:
```
Comment (catherineluse) [inline content/rancher/v2.x/en/cluster-admin/upgrade-cert-manager/_index.md]: November

**Comment by drpebcak** [comment]
Looks good to me.. are we going to provide catalog specific instructions for people who installed cert-manager from the catalog? And also, are we going to remove the old version of cert-manager from the library?

**Comment by dramich** [comment]
@drpebcak Users should be able to just upgrade like a normal app once https://github.com/rancher/charts/pull/206 is merged.

**Comment by oskapt** [comment]
LGTM, with one question. There's a [new solver format in 0.8](https://docs.cert-manager.io/en/latest/tasks/upgrading/upgrading-0.7-0.8.html) where the config moves from the cert to the Issuer/ClusterIssuer and requires that the cluster admin move `http01` from `spec.acme` to `spec.acme.solvers` while also removing it from `spec.acme.config` in the certificate itself. Not migrating to that config now won't break anything, but it's deprecated and will be removed before 1.0. These instructions skip from 0.5 to 0.9 and if we don't call out that change, users might be surprised when it bites them in the future.

File: content/rancher/v2.x/en/cluster-admin/upgrade-cert-manager-airgap/_index.md [hunk 5]
```diff
@@ -0,0 +1,92 @@
+---
+title: Upgrade Cert-manager Airgap
+weight: 2040
+---
+
+[Let's Encrypt will be blocking cert-manager instances older than 0.8.0 starting November 1st 2019.](https://community.letsencrypt.org/t/blocking-old-cert-manager-versions/98753) In order to upgrade cert-manager to the newer version follow these instructions:
+
+>**Note:** The namespaces used in these instructions depends on the namespace cert-manager is currently installed in. If it is in kube-system use that in the instructions below. You can verify by running `kubectl get pods --all-namespaces` and checking which namespace the cert-manager-* pods are listed in. Do not change the namespace cert-manager is running in or this can cause issues.
```
Comment (oskapt) [inline content/rancher/v2.x/en/cluster-admin/upgrade-cert-manager-airgap/_index.md]: Namespace should be singular: 

> The namespace used in these instructions...

**Comment by cjellick** [comment]
We need a few warnings:
- the big 1.0 breaking change 
- cert-manager rolling disabling

**Comment by cjellick** [comment]
Add a note calling out the data migration that will need to occur. This should referene their migration guide:
https://docs.cert-manager.io/en/latest/tasks/upgrading/upgrading-0.7-0.8.html#performing-an-incremental-switch-to-the-new-format

**Comment by cjellick** [comment]
at top of upgrade pages, add mini summary of the two things we are doing.

**Comment by cjellick** [comment]
I took this off dan's plate and made the desired updates here: https://github.com/rancher/docs/pull/1713

**Comment by cjellick** [comment]
Im going to close in favor of mine

**Pull request closed without merging**